lib/Store.js

```diff
@@ -6,7 +6,8 @@ var _classCallCheck = function (instance, Constructor) { if (!(instance instance
 
 var EventEmitter = require("events").EventEmitter;
 var assign = require("object-assign");
-var iv = require("invariant");
+var SimpleConsole = require("simple-console");
+var con = process.env.NODE_ENV === "production" ? new SimpleConsole(null) : new SimpleConsole();
```

Comment:
GLOBAL: I'd move all of this to one file with a singleton to simplify all of this like `lib/console.js`

```js
module.exports = process.env.NODE_ENV === "production" ? new SimpleConsole(null) : new SimpleConsole();
```

And then elsewhere:

```js
var con = require('./lib/console');
```